test/Escrow.t.sol

```diff
@@ -53,6 +53,59 @@ contract EscrowContractTest is Test {
 
     bool runIntegrationTests = false;
 
+
+    event Deposit(
+        address indexed sender,
+        address indexed receiver,
+        uint256 amount
+    );
+    event UnassignedDeposit(
+        address indexed depositor,
+        address indexed sender,
+        uint256 amount
+    );
+    event DepositAssigned(
+        address indexed sender,
+        address indexed receiver,
+        uint256 amount
+    );
+    event Redeem(
+        address indexed sender,
+        address indexed receiver,
+        address indexed allocationID,
+        uint256 expectedAmount,
+        uint256 actualAmount
+    );
+    event Thaw(
+        address indexed sender,
+        address indexed receiver,
+        uint256 amount,
+        uint256 totalAmountThawing,
+        uint256 thawEndTimestamp
+    );
+    event CancelThaw(address indexed sender, address indexed receiver);
+    event ThawSigner(
+        address indexed sender,
+        address indexed authorizedSigner,
+        uint256 thawEndTimestamp
+    );
+    event CancelThawSigner(
+        address indexed sender,
+        address indexed authorizedSigner,
+        uint256 thawEndTimestamp
+    );
+    event RevokeAuthorizedSigner(
+        address indexed sender,
+        address indexed authorizedSigner
+    );
+    event Withdraw(
+        address indexed sender,
+        address indexed receiver,
+        uint256 amount
+    );
+    event AuthorizeSigner(address indexed signer, address indexed sender);
+    event Transfer(address indexed from, address indexed to, uint256 value);
+
```

Comment:
Is it possible to import the events instead of redefining them? It seems like this could make maintenance difficult.

Reply:
Unfortunately there isn't a clean way of doing it. Best thing I can think of is to define an interface for all the events then inherit it on both the contract and the test contract, but i'm not too happy with that solution. 

Foundry seems to recommend just re declaring the events: https://book.getfoundry.sh/cheatcodes/expect-emit, which I agree is also not good due to maintenance and duplicate code.